build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml

```diff
@@ -60,6 +60,13 @@ jobs:
         sourceFolder: '$(Build.SourcesDirectory)\Artifacts\drop'
         Contents: '**\Microsoft.UI.Xaml*.appx'
 
+    # Copy the CBS packages into the same location in the artifact drop to be republished at the end of the job.
```

Comment:
Optional: We can probably re-use the previous step by updating the 'Contents' expression. You can probably have multiple expressions separated by semicolons or commas or something. I am not certain of the exact syntax.s

Reply:
I will keep it as it is